src/components/App/ExpressRecorder.tsx

```diff
@@ -670,6 +670,9 @@ export class ExpressRecorder extends Component<ExpressRecorderProps, State> {
                             onClick={this.handleCancelClick}
                             tabIndex={0}
                             ref={node => (this.cancelButtonRef = node as HTMLMediaElement)}
+                            aria-label={this.translator.translate(
+                                "recording will start in a three seconds delay"
```

Comment:
@odedm-kaltura this doesn't seem to be the correct place/timing for this aria-label, it is not related to this button.
I think the right place is the aria-label for the "start recording" button, as an extension to the text there.